definitions/json/AuthorisationCaseField/AuthorisationCaseField-WA.json

```
@@ -539,6 +599,30 @@
     "UserRole": "et-acas-api",
     "CRUD": "R"
   },
+  {
+    "CaseTypeId": "ET_EnglandWales",
+    "CaseFieldID": "etICHearingNotListedListForFinalHearingUpdated",
+    "UserRole": "caseworker-employment-englandwales",
+    "CRUD": "CRUD"
+  },
```

Permissions for et-judge
```suggestion
  {
    "CaseTypeId": "ET_EnglandWales",
    "CaseFieldID": "etICHearingNotListedListForFinalHearingUpdated",
    "UserRole": "caseworker-employment-etjudge",
    "CRUD": "CRUD"
  },
  {
    "CaseTypeId": "ET_EnglandWales",
    "CaseFieldID": "etICHearingNotListedListForFinalHearingUpdated",
    "UserRole": "caseworker-employment-englandwales",
    "CRUD": "CRUD"
  },
```